### PR TITLE
K8SPG-416 add hba custom config

### PR DESCRIPTION
**Pull request by spron-in**

### CODE REVIEW

File: charts/pg-db/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: pg-db
 description: 'A Helm chart to deploy the PostgreSQL database with the Percona Operator for PostgreSQL'
 type: application
-version: 2.2.2
+version: 2.2.4
```
Comment (tplavcic) [inline charts/pg-db/Chart.yaml:5]: Why increase two versions?

File: charts/pg-db/templates/cluster.yaml [hunk 2]
```diff
@@ -357,13 +357,14 @@ spec:
...
-   patroni:
-     dynamicConfiguration:
-       postgresql:
-         parameters:
-           max_parallel_workers: {{ .Values.patroni.dynamicConfiguration.postgresql.parameters.max_parallel_workers }}
-           max_worker_processes: {{ .Values.patroni.dynamicConfiguration.postgresql.parameters.max_worker_processes }}
-           shared_buffers: {{ .Values.patroni.dynamicConfiguration.postgresql.parameters.shared_buffers }}
-           work_mem: {{ .Values.patroni.dynamicConfiguration.postgresql.parameters.work_mem }}
- {{- end }}
+{{- if .Values.patroni}}
```
Comment (tplavcic) [inline charts/pg-db/templates/cluster.yaml:360]: add space before `}}` so it's consistent

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: charts/pg-db/Chart.yaml (hunk 1 above)
Comment (nmarukovich) [reply charts/pg-db/Chart.yaml:5]: we have 2.3.0 in 
https://github.com/percona/percona-helm-charts/pull/232/files#diff-64472c1e36d35d9916e54a8a05dbc0b508cb4268e822ee5ad69846e44f28989c

**Comment by nmarukovich** [comment]
I tested. It works for me. @spron-in could you please fix conflicts?

**Review by hors — APPROVED** [review]

**Review by nmarukovich — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**